src/libopenrave-core/colladaparser/colladareader.cpp

```diff
@@ -296,7 +296,7 @@ class ColladaReader : public daeErrorHandler
         if (find(_vOpenRAVESchemeAliases.begin(), _vOpenRAVESchemeAliases.end(), urioriginal.scheme()) !=
             _vOpenRAVESchemeAliases.end()) {
             if (urioriginal.path().empty()) {
-                return nullptr;
+                return std::string();
```

Comment:
so this tried to construct std::string from c-string, which is not a compile error but will cause a runtime error.

Reply:
thank you!